tools/flub/src/commands/bump.ts

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// import { getResolvedFluidRoot } from "@fluidframework/build-tools/src/common/fluidUtils";
```

Comment:
Should these commented-out imports be removed?

Reply:
Yeah, I have some more changes in this PR that I need to clean up.